components/apim-analytics/org.wso2.analytics.solutions.apim.analytics/widgets/APIMApiCreated/src/APIMApiCreated.jsx

```diff
@@ -0,0 +1,372 @@
+ * @type {string}
+ */
+const language = (navigator.languages && navigator.languages[0]) || navigator.language || navigator.userLanguage;
+
+/**
+ * Language without region code
+ */
+const languageWithoutRegionCode = language.toLowerCase().split(/[_-]+/)[0];
+
+/**
+ * Create React Component for APIM Api Created
+ * @class APIMApiCreated
+ * @extends {Widget}
+ */
+class APIMApiCreated extends Widget {
+    /**
+     * Creates an instance of APIMApiCreated.
+     * @param {any} props @inheritDoc
+     * @memberof APIMApiCreated
+     */
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            width: this.props.glContainer.width,
+            height: this.props.glContainer.height,
+            totalCount: 0,
+            weekCount: 0,
+            localeMessages: null,
+        };
+
+        this.styles = {
+            headingWrapper: {
+                height: '10%',
+                margin: 'auto',
+                paddingTop: '15px',
+                width: '90%',
+            },
+            cIconWrapper: {
+                float: 'left',
+                width: '40%',
+                height: '62%',
+            },
+            cIcon: {
+                display: 'block',
+                margin: 'auto',
+                marginTop: '25%',
+            },
+            dataWrapper: {
+                float: 'left',
+                width: '60%',
+                height: '50%',
+                paddingTop: '8%',
+            },
+            weekCount: {
+                margin: 0,
+                marginTop: '5%',
+                color: 'rgb(135,205,223)',
+                letterSpacing: 1,
+                fontSize: '80%',
+            },
+            typeText: {
+                textAlign: 'left',
+                fontWeight: 'normal',
+                margin: 0,
+                display: 'inline',
+                marginLeft: '3%',
+                letterSpacing: 1.5,
+                fontSize: 'small',
+            },
+            icon: {
+                position: 'absolute',
+                bottom: '13%',
+                right: '8%',
+            },
+            loadingIcon: {
+                margin: 'auto',
+                display: 'block',
+            },
+        };
+
+        this.props.glContainer.on('resize', () => this.setState({
+            width: this.props.glContainer.width,
+            height: this.props.glContainer.height,
+        }));
+
+        this.assembleweekQuery = this.assembleweekQuery.bind(this);
+        this.assembletotalQuery = this.assembletotalQuery.bind(this);
+        this.handleWeekCountReceived = this.handleWeekCountReceived.bind(this);
+        this.handleTotalCountReceived = this.handleTotalCountReceived.bind(this);
+        this.loadLocale = this.loadLocale.bind(this);
+    }
+
+    componentDidMount() {
+        const locale = languageWithoutRegionCode || language;
+        this.loadLocale(locale).catch(() => {
+            this.loadLocale().catch(() => {
```

Comment:
Why `loadLocale` is called twice here?
Also, lets add a `console.error` in Promise `catch` when there is no error message is shown to the user.